indico/worker/Dockerfile

```diff
@@ -1,58 +1,40 @@
-FROM centos:centos7
+FROM python:2.7-buster
 
-ENV INDICO_VIRTUALENV="/opt/indico/.venv/bin" INDICO_CONFIG="/opt/indico/etc/indico.conf"
+ENV INDICO_VIRTUALENV="/opt/indico/.venv" INDICO_CONFIG="/opt/indico/etc/indico.conf"
 
 ARG KRB_CACHE_DIR='/var/run/keytab'
-ARG pip="${INDICO_VIRTUALENV}/pip"
-ARG use_xrootd=""
+ARG pip="${INDICO_VIRTUALENV}/bin/pip"
 ARG tag="latest"
 
 ENV KRB_CACHE_DIR ${KRB_CACHE_DIR}
 
 USER root
 
-RUN yum update -y;
-RUN yum install -y epel-release;
-RUN yum install -y texlive-xetex
-RUN yum localinstall -y https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-3.noarch.rpm
-RUN yum update -y
-RUN yum install -y postgresql96
-RUN mkdir -p --mode=775 /opt/indico/{etc,tmp,log,assets,cache,archive,static}
-
-RUN if [ -n "${use_xrootd}" ]; then \
-    yum update -y && \
-    yum install -y xrootd-client xrootd-client-devel xrootd-python; \
-fi
-RUN yum install -y vim less python-virtualenv gcc gettext
-RUN yum clean all
+RUN apt-get update
+RUN apt-get install -y texlive-xetex libpq-dev postgresql-client
 
-COPY passwd.template /tmp/
+RUN ["/bin/bash", "-c", "mkdir -p --mode=775 /opt/indico/{etc,tmp,log,assets,cache,archive}"]
 
-RUN if [ -z "${use_xrootd}" ]; then \
-    virtualenv /opt/indico/.venv; \
-else \
-    virtualenv /opt/indico/.venv --system-site-packages; \
-fi
+RUN apt-get install -y vim less python-virtualenv gcc gettext
+RUN apt-get clean
```

Comment:
```
RUN set -ex && \
    apt-get update && \
    apt-get -y install ... && \
    apt-get clean
```

Otherwise you get multiple layers and don't benefit from cleaning up at all

Reply:
Good point